Dockerfile

```
@@ -438,21 +438,23 @@ RUN apt-get update \
 COPY scripts/docker/install_mysql.sh /scripts/docker/install_mssql.sh /scripts/docker/
 
 # fix permission issue in Azure DevOps when running the scripts
-RUN chmod a+x /scripts/docker/install_mysql.sh && \
-    /scripts/docker/install_mysql.sh prod && \
-    chmod a+x /scripts/docker/install_mssql.sh && \
-    /scripts/docker/install_mssql.sh && \
-    adduser --gecos "First Last,RoomNumber,WorkPhone,HomePhone" --disabled-password \
-           --quiet "airflow" --uid "${AIRFLOW_UID}" --gid "0" --home "${AIRFLOW_USER_HOME_DIR}" && \
+RUN chmod a+x /scripts/docker/install_mysql.sh \
+    && sync \
+    && /scripts/docker/install_mysql.sh prod \
+    && chmod a+x /scripts/docker/install_mssql.sh \
+    && sync \
```

Perhaps?

```suggestion
RUN chmod a+x /scripts/docker/install_mysql.sh /scripts/docker/install_mssql.sh \
    && sync \
    && /scripts/docker/install_mysql.sh prod \
```

(Either way avoiding two `sync`s probably worth it.

---

Definitely worth it! 

I fixed it and also synchronized and updated comments explaining why we are doing it at all (running chmod+x is for Azure where +x is not carried via filesystem - which is lame but we have to do it for Azure) as this is far from obvious why we are doing it at all.